power-platform/admin/environment-groups.md

```diff
@@ -27,6 +27,7 @@ Admins can apply various rules to groups to govern many environments in bulk, re
 - [Solution checker enforcement](managed-environment-solution-checker.md)
 - [Usage insights](managed-environment-usage-insights.md)
 - [Extended backup retention](backup-restore-environments.md)
+- [Move data across regions](geographical-availability-copilot.md)
```

Comment:
we should change the sentence above which references "first release" and "six key capabilities" - @mikferland-msft suggestions?

Reply:
also there's another reference to "six" below and it says "which are the same capabilities found in [Managed Environment](managed-environment-overview.md)" which is not true anymore with this one

Reply:
Line 22: Replace "In this first release..." with "Specifically, the following rules are available within each environment group:".
Linne 24: Place "AI-generated descriptions" and "Move data across regions" as the first two elements in the list.

Reply:
updated